planetary-social/nos
# Open profiles when tapping on a NIP-05 username

## Issues covered
#1087 

## Description
When parsing a note, instead of switching nip-05 usernames like linda@nos.social to njpump links, I replace them with urls of the form @linda@nos.social (this is similar to @npub1 urls). So, when opening this links, we check if it an npub or hex, if it is not, we try to fetch the npub from the nip-05 username.

## How to test
1. Navigate to Discover screen
2. Search npub1ey39gym4zlppcsvquqhv0cujnsn6uwuu9z9f4sxkzp5vjy8gfa9sprdq23
3. Tap on linda@nos.social
4. The new account of Linda is shown.

## Screenshots/Video


https://github.com/planetary-social/nos/assets/1703242/78286e62-87f4-4079-983f-9641f4dedc98
